docs/static/includes/module-features/bicepFeatures.csv

```diff
@@ -1,93 +1,93 @@
 #,Module,RBAC,Locks,Tags,Diag,PE,PIP
-1,res/storage/storage-account,✅,✅,✅,✅,✅,
-2,res/network/route-table,✅,✅,✅,,,
-3,res/network/vpn-gateway,,✅,✅,,,
-4,res/network/load-balancer,✅,✅,✅,✅,,
-5,res/network/private-dns-zone,✅,✅,✅,,,
-6,res/network/express-route-circuit,✅,✅,✅,✅,,
-7,res/network/network-interface,✅,✅,✅,✅,,
-8,res/network/dns-resolver,✅,✅,✅,,,
-9,res/network/local-network-gateway,✅,✅,✅,,,
-10,res/network/dns-forwarding-ruleset,✅,✅,✅,,,
-11,res/network/application-security-group,✅,✅,✅,,,
-12,res/network/private-endpoint,✅,✅,✅,,,
-13,res/network/dns-zone,✅,✅,✅,,,
-14,res/network/ddos-protection-plan,✅,✅,✅,,,
-15,res/network/virtual-network-gateway,✅,✅,✅,✅,,
-16,res/network/network-security-group,✅,✅,✅,✅,,
-17,res/network/bastion-host,✅,✅,✅,✅,,✅
-18,res/network/public-ip-prefix,✅,✅,✅,,,
-19,res/network/connection,,✅,✅,,,
-20,res/network/express-route-gateway,✅,✅,✅,,,
-21,res/network/front-door,✅,✅,✅,✅,,
-22,res/network/firewall-policy,,,✅,,,
-23,res/network/ip-group,✅,✅,✅,,,
-24,res/network/vpn-site,✅,✅,✅,,,
-25,res/network/virtual-network,✅,✅,✅,✅,,
-26,res/network/trafficmanagerprofile,✅,✅,✅,✅,,
-27,res/network/public-ip-address,✅,✅,✅,✅,,
-28,res/network/front-door-web-application-firewall-policy,✅,✅,✅,,,
-29,res/network/nat-gateway,✅,✅,✅,,,
-30,res/insights/private-link-scope,✅,✅,✅,,✅,
-31,res/insights/data-collection-endpoint,✅,✅,✅,,,
-32,res/insights/data-collection-rule,✅,✅,✅,,,
-33,res/insights/component,✅,,✅,✅,,
-34,res/insights/webtest,✅,✅,✅,,,
-35,res/insights/activity-log-alert,✅,,✅,,,
-36,res/insights/metric-alert,✅,,✅,,,
-37,res/insights/action-group,✅,,✅,,,
-38,res/insights/scheduled-query-rule,✅,,✅,,,
-39,res/insights/diagnostic-setting,,,,,,
-40,res/container-service/managed-cluster,✅,✅,✅,✅,,
-41,res/sql/server,✅,✅,✅,,✅,
-42,res/health-bot/health-bot,✅,✅,✅,,,
-43,res/maintenance/maintenance-configuration,✅,✅,✅,,,
-44,res/desktop-virtualization/application-group,✅,✅,✅,✅,,
-45,res/desktop-virtualization/scaling-plan,✅,✅,✅,✅,,
-46,res/databricks/workspace,✅,✅,✅,✅,✅,
-47,res/databricks/access-connector,✅,✅,✅,,,
-48,res/logic/workflow,✅,✅,✅,✅,,
-49,res/web/static-site,✅,✅,✅,,✅,
-50,res/web/serverfarm,✅,✅,✅,✅,,
-51,res/web/site,✅,✅,✅,✅,✅,
+1,res/data-protection/backup-vault,✅,✅,✅,,,
```

Comment:
For some reason this isn't alphabetical - not was it before. This needs to be fixed